sdcm/cluster_aws.py

```diff
@@ -551,7 +551,12 @@ def _get_ipv6_ip_address(self) -> Optional[str]:
         return self._instance.network_interfaces[0].ipv6_addresses[0]["Ipv6Address"]
 
     def _refresh_instance_state(self):
-        raise NotImplementedError()
+        self._wait_public_ip()
+        public_ipv4_addresses = [self._instance.public_ip_address]
+        private_ipv4_addresses = [self._instance.private_ip_address]
+        if self._eth1_private_ip_address:
+            private_ipv4_addresses += [self._eth1_private_ip_address]
```

Comment:
isn't it better to `private_ipv4_addresses.append(self._eth1_private_ip_address)` ?

Reply:
it's equivalent, maybe a bit slower.

for for the once test case we are using the extra network interface... it's not that important